backend/models/form.py

```diff
@@ -12,29 +12,29 @@ class Form(BaseModel):
     """Schema model for form."""
 
     id: str = Field(alias="_id")
-    features: t.List[str]
-    questions: t.List[Question]
+    features: list[str]
+    questions: list[Question]
     name: str
     description: str
 
     class Config:
         allow_population_by_field_name = True
 
     @validator("features")
-    def validate_features(cls, value: t.List[str]) -> t.Optional[t.List[str]]:
+    def validate_features(cls, value: list[str]) -> t.Optional[list[str]]:
         """Validates is all features in allowed list."""
         # Uppercase everything to avoid mixed case in DB
         value = [v.upper() for v in value]
-        allowed_values = list(v.value for v in FormFeatures.__members__.values())
-        if not all(v in allowed_values for v in value):
+        allowed_values = [v.value for v in FormFeatures.__members__.values()]
+        if any(v not in allowed_values for v in value):
             raise ValueError("Form features list contains one or more invalid values.")
 
         if FormFeatures.COLLECT_EMAIL in value and FormFeatures.REQUIRES_LOGIN not in value:  # noqa
             raise ValueError("COLLECT_EMAIL feature require REQUIRES_LOGIN feature.")
 
         return value
 
-    def dict(self, admin: bool = True, **kwargs: t.Dict) -> t.Dict[str, t.Any]:
+    def dict(self, admin: bool = True, **kwargs: t.Any) -> dict[str, t.Any]:
```

Comment:
Why would kwargs be `t.Any`? Wouldn't it make more sense to be `dict[str, t.Any]`?

Reply:
cc @decorator-factory since I think this is a refactor.

Reply:
@jb3 kwargs are hinted by the type of the values: https://mypy.readthedocs.io/en/stable/cheat_sheet_py3.html#when-you-re-puzzled-or-when-things-are-complicated

Reply:
Just like `*args` are not hinted as `tuple[str, ...]`, but as `str`

Reply:
Ah interesting. Thanks!